Makefile

```diff
@@ -143,7 +143,7 @@ operator_register_with_eigen_layer:
 
 operator_mint_mock_tokens:
 	@echo "Minting tokens"
-	. ./scripts/mint_mock_token.sh $(CONFIG_FILE) 1000
+	. ./scripts/mint_mock_token.sh $(CONFIG_FILE) 100000000000000000
```

Comment:
Should this be changed back to 1000?

Reply:
No, 1000 wei is too small value to be shown in operator stakes. Now its 0.1 eth